netbox/utilities/filters.py

```diff
@@ -63,6 +63,11 @@ class TreeNodeMultipleChoiceFilter(django_filters.ModelMultipleChoiceFilter):
     Filters for a set of Models, including all descendant models within a Tree.  Example: [<Region: R1>,<Region: R2>]
     """
     def filter(self, qs, value):
+        if 'null' in value:
```

Comment:
Convention requires that we refer to `settings.FILTERS_NULL_CHOICE_VALUE` for the null value. This is currently the string `"null"` but it could change in the future.